core/src/main/scala/org/apache/spark/Accumulator.scala

```diff
@@ -26,7 +26,7 @@ package org.apache.spark
  *
  * An accumulator is created from an initial value `v` by calling
  * [[SparkContext#accumulator SparkContext.accumulator]].
- * Tasks running on the cluster can then add to it using the [[Accumulable#+= +=]] operator.
+ * Tasks running on the cluster can then add to it using the `+=` operator.
```

Comment:
After this PR it still prints the same.

- Scala

  <img width="97" alt="2016-11-26 12 51 56" src="https://cloud.githubusercontent.com/assets/6477701/20637848/2d670926-b3d7-11e6-8665-a9f3852545c2.png">

- Java

  <img width="125" alt="2016-11-26 12 51 38" src="https://cloud.githubusercontent.com/assets/6477701/20637849/322b675e-b3d7-11e6-925b-a9160f06bbc8.png">